beacon-chain/state/v2/deprecated_setters.go

```diff
@@ -5,16 +5,6 @@ import (
 	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
 )
 
-// SetPreviousEpochAttestations is not supported for HF1 beacon state.
-func (*BeaconState) SetPreviousEpochAttestations(_ []*ethpb.PendingAttestation) error {
-	return errors.New("SetPreviousEpochAttestations is not supported for hard fork 1 beacon state")
-}
-
-// SetCurrentEpochAttestations is not supported for HF1 beacon state.
-func (*BeaconState) SetCurrentEpochAttestations(_ []*ethpb.PendingAttestation) error {
-	return errors.New("SetCurrentEpochAttestations is not supported for hard fork 1 beacon state")
-}
```

Comment:
Not used anywhere except tests.

Reply:
Not sure if we can actually remove these methods, I think an interface breaks if we do not use them, surprised that no tests actually fail in that case.

Reply:
They are not a part of the interface. They are unexported for phase 0.

Reply:
So it was exported for no reason...